experiment_2.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import statistics
+
+from pathlib import PurePosixPath
```

Comment:
```suggestion
from pathlib import Path
```
`Path` is a universal class which should be used always